plugins/admin/core/root.go

```diff
@@ -31,21 +31,17 @@ import (
 var cfgFile string
 
 // rootCmd represents the base command when called without any subcommands
-
 func NewAdminCommand(params ...pkg.AdminParams) *cobra.Command {
 	p := &pkg.AdminParams{}
 	p.Initialize()
 
 	rootCmd := &cobra.Command{
 		Use:   "kn\u00A0admin",
 		Short: "A plugin of kn client to manage Knative",
-		Long: `kn admin: a plugin of kn client to manage Knative for administrators.
+		Long:  `kn admin: a plugin of kn client to manage Knative for administrators`,
 
-For example:
-kn admin domain set - to set Knative route domain
-kn admin registry add - to add registry with credentials
-kn admin autoscaling update - to manage autoscaling config
-`,
+		// disable printing usage when error occurs
+		SilenceUsage: true,
```

Comment:
As Grace explained in above comment, we use ```\u00A0``` to show:
```
Usage:
  kn admin [command]
```
Without the unicode, it will show as:
```
Usage:
  kn [command]
```

Reply:
Wow, that's interesting. Can you please add a UT test for this. Thanks.

Reply:
ok, no problem, I will add UT for it.